amulet/sentry.py

```diff
@@ -120,10 +120,11 @@ def upload_scripts(self):
 
         # copy one at a time b/c `juju scp -r` doesn't work (currently)
         for f in glob.glob(os.path.join(source, '*')):
-            cmd = "juju scp {} {}:{}".format(
-                os.path.join(source, f),
-                self.info['unit_name'], dest)
-            subprocess.check_call(cmd.split())
+            src = os.path.join(source, f)
+            if os.path.isfile(src):
```

Comment:
If there are valid directories in `/tmp/amulet`, scp -r would be another option.  The comment notes that this doesn't work, but it does (at least with juju2).  Syntax is like this:

`juju scp -- -r {} {}:{}...`

If the only dir in `/tmp/amulet` is `__pycache__`, then sure, let's glob and only scp files.